### ISSUE TITLE
[Report]

### ISSUE THREAD

**Issue by skeedZ** [holoiso-eol/holoiso#842]
## Your Issue:
AMD iGPU and Nvidia dGPU
Hello,
I know Nvidia GPU compatibility isn't any good for gamepad ui but I have a question regarding the compatibility of those 2.
If I would be using and AMD Radeon RX Vega 7 for the display of the OS and install the drivers for Nvidia, could it possible the games just to run on the Nvidia or would be any more issues?
Thank you and best regards!
```

## HP Pavilion Gaming 15
```
CPU:  AMD Ryzen 5600H
GPU: AMD Radeon RX Vega 7 + Nvidia RTX 3050 TI
RAM: 16GB
```

## Your OS version (`cat /etc/os-release | grep VARIANT_ID | cut -d '"' -f 2`):
```
No OS Just question.
```

## (Only for SteamOS Session loading issues) Have you tried launching `gamescope-session` from terminal? If yes, provide a log.
```
No OS Just question.*
```

## Have you updated your installation recently?
```
No OS Just question.
```

**Comment by Halornek** [comment]
Just throwing my two cents in here, not an expert by far so if anyone wants to correct me please go ahead.

Gamemode basically can not work properly with an Nvidia GPU. "Gamemode" was made specifically for the Steam Deck using a custom compositor developed by Valve under the Wayland protocol (AFAIK). Valve hasn't bothered to add support for anything except AMD. The reason Intel GPUs can work is because Intel and AMD share the same userspace drivers, `mesa`.

If you can get desktop mode to boot using the dedicated Nvidia GPU, you could run games there, even via Big Picture mode, but this would be similar to running Big Picture on a standard KDE Plasma desktop or even Windows. The main concern here is that desktop mode uses a X11 instead of Wayland and does not properly support GPU switching (I/E, rendering the desktop with the Vega iGPU and games with the dedicated Nvidia GPU). You could work around this with via [prime](https://wiki.archlinux.org/title/PRIME) and Steam Command line arguments if you really wanted to.

Long story short, you could potentially get this to work with some jank and hacking around a bit, but it's unlikely to be a good experience. HoloISO is already doing a bit of hacking around and you have a lot of points for failure adding more to it.

Hope this helps. Let me know if you have any questions.
